src/Sylius/Bundle/InventoryBundle/DependencyInjection/Configuration.php

```diff
@@ -34,7 +34,11 @@ public function getConfigTreeBuilder(): TreeBuilder
             ->addDefaultsIfNotSet()
             ->children()
                 ->scalarNode('driver')->defaultValue(SyliusResourceBundle::DRIVER_DOCTRINE_ORM)->end()
-                ->scalarNode('checker')->defaultValue('sylius.availability_checker.default')->cannotBeEmpty()->end()
+                ->scalarNode('checker')
+                    ->setDeprecated('sylius/inventory-bundle', '1.13', 'The "%path%.%node%" parameter is deprecated and will be removed in 2.0.')
```

Comment:
```suggestion
                    ->setDeprecated('sylius/inventory-bundle', '1.13', 'The "%path%.%node%" is deprecated and will be removed in 2.0.')
```
Think it'll suffice given the context of where it gets shown.